styles.css

```diff
@@ -191,3 +191,30 @@ body {
   border-radius: 15px;
   background: skyblue;
 }
+
+@media (max-width: 800px){
+  .container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+  }
+
+  .flagsSelector{
+    rotate: 270deg;
```

Comment:
Responsive design shoudn't be done with these kind if styles, we should uodate the width, height and flexbox properties in order to do that.